sphinx/util/logging.py

```diff
@@ -131,6 +132,11 @@ def verbose(self, msg: str, *args: Any, **kwargs: Any) -> None:
         self.log(VERBOSE, msg, *args, **kwargs)
 
     def process(self, msg: str, kwargs: Dict) -> Tuple[str, Dict]:  # type: ignore
+        if 'exc_info' not in kwargs:
+            exc_info = sys.exc_info()
+            if exc_info != (None, None, None):
+                kwargs['exc_info'] = True
```

Comment:
I'd still probably argue this is up to the caller to decide, and therefore this change isn't necessary - there are cases when the exception is just not interesting, such as:
```python
try:
    val = some_dict[key]
except KeyError:
    log.info("Unable to find data for {}".format(key))
```
Seeing the exception here is just noise.

Reply:
I'd like to give exc_info flag to all loggers. But it seems not needed. Okay, withdrawing.

Reply:
I think the rest of this patch is useful, I'd encourage you to withdraw only these four lines.

Reply:
I feel too much to add it only for one exc_info call. Do you have a plan to add it more?